src/silkaj.py

```diff
@@ -71,6 +73,11 @@ def manage_cmd(ep, c):
         difficulties(ep)
 
     elif c.subcmd == "network":
+        from commands import set_network_sort_keys
```

Comment:
Yep, `argparse` could be a good candidate.
`CommandLine` is no more maintained since one year.